_works/2025-maybauer-finn/02-titles-and-research-questions.md

```diff
@@ -0,0 +1,35 @@
+# Titel und Forschungsfrage
+
+Bitte notieren Sie mindestens 10 mögliche Titel der Arbeit und mindestens 5 mögliche Forschungsfragen/Zielsetzungen in dieses Dokument.
+
+## Titel
+
+### Iteration 0
+
+> [!NOTE]  
+> Titel generiert über ChatGPT 4o. Prompt: "Generiere 10 mögliche Titel für eine Masterarbeit in der Medieninformatik mit folgendem Expose: <[Inhalt des Exposes](03-expose.md)"
+
+* Barrierefreiheit in Unternehmen: Ein Reifegrad-Ansatz zur Umsetzung der EU-Norm EN 301 549
+* Von Compliance zu Inklusion: Strategien zur Messung und Steigerung der Barrierefreiheitsreife
+* Reifegradmodelle für barrierefreie digitale Produkte: Herausforderungen und Chancen
+* Barrierefreiheit als Prozess: Entwicklung eines Werkzeuges zur Maturity-Messung
+* Inklusive Digitalisierung: Methoden zur Steigerung der Barrierefreiheit in Organisationen
+* Workshop oder Web-Tool? Ansätze zur Prozessoptimierung in der Barrierefreiheit
+* Maturity-Modell-Ansätze für die Barrierefreiheit: Eine Fallstudie
+* Barrierefreiheits-Compliance: Werkzeuge und Methoden zur organisatorischen Vorbereitung
+* Messbare Inklusion: Ein Reifegradmodell zur Umsetzung der EU-Norm EN 301 549
+* Digitale Barrierefreiheit im Fokus: Entwicklung einer Methodik zur Prozessverbesserung
+
+### Iteration 1
+
+* Barrierefreiheit in Unternehmen: Messung und Steigerung der Reife durch Maturity-Modelle
+* Implementierung der EU-Norm EN 301 549: Reifegradmodelle als Werkzeug zur Stärkung der Barrierefreiheit in Organisationen
+* Steigerung der Barrierefreiheitsreife: Anwendung von Maturity-Modellen in softwareentwickelnden Organisationen
+
+## Forschungsfragen/Zielsetzungen
+
+* Wie kann der Reifegrad eines softwareentwickelnden Unternehmens hinsichtlich der Barrierefreiheit gemessen werden?
+* Wie lässt sich die gemessene Reife der Barrierefreiheit in Organisationen durch Maturity-Modelle verbessern?
+* Welche Herausforderungen ergeben sich bei der Anwendung von Maturity-Modellen zur Steigerung der Barrierefreiheit in Unternehmen?
+* Welche Methoden und Werkzeuge können Organisationen bei der Vorbereitung auf die EU-Norm EN 301 549 unterstützen?
```

Comment:
Ich habe meine Zeitplan konkretisiert. Es ist fraglich in weit hier eine "Vorbereitung" möglich ist, da die Norm am 28. Juni in Kraft tritt.